uaclient/entitlements/fips.py

```diff
@@ -42,6 +45,14 @@ def application_status(self) -> "Tuple[status.ApplicationStatus, str]":
             "Reboot to FIPS kernel required",
         )
 
+    def enable(self, *, silent_if_inapplicable: bool = False) -> bool:
+        """FIPS entitlements need to set reboot-required flag upon success"""
+        enabled = super().enable(silent_if_inapplicable=silent_if_inapplicable)
+        if enabled:
+            if not os.path.exists(REBOOT_REQUIRED_FILE):
+                util.write_file(REBOOT_REQUIRED_FILE, "")
```

Comment:
My `/var/run/reboot-required` currently contains:

```
*** System restart required ***
```

and I also have `/var/run/reboot-required.pkgs`, which contains:

```
libc6
libc6
```

Should we be inserting that string into the file, rather than empty content? Should we be adding `ubuntu-advantage-tools` or `fips` or something to the `.pkgs` file?

Reply:
@OddBloke can you please ask in #ubuntu-devel the correct usage of /var/run/reboot-required?

Reply:
@powersj It occurs to me: as it's the kernel that requires a reboot to use, why isn't the kernel packaging writing this out for us?

Reply:
Aaaand the answer is: it is!

```
$ cat /var/run/reboot-required
*** System restart required ***
$ cat /var/run/reboot-required.pkgs 
libssl1.0.0
linux-image-4.4.0-1002-fips
linux-base
linux-base
```

Reply:
I pinged in IRC, but to make sure it doesn't get lost: I still think we can drop this code.